README.md

```diff
@@ -1,25 +1,74 @@
 # darkforest-local
 
-The Dark Forest client deploy process is set up for players to easily fork the client and connect to the mainnet game, but that design decision currently makes it difficult to run a playable version of the game locally. This repository provides a setup for running a local game with just a few steps.
-
-This repo uses submodules to pair the [Dark Forest Ethereum backend](https://github.com/darkforest-eth/eth) with the [Dark Forest TypeScript frontend](https://github.com/darkforest-eth/client) so you can launch a local game.
-
-## Install
-
-- Clone with submodules `git clone --recurse-submodules https://github.com/projectsophon/darkforest-local` or if you've already cloned and want to update the module `git submodule update --init --recursive`
-- Install [Yarn](https://classic.yarnpkg.com/en/docs/install)
-- Run `yarn` at the top level to automatically install all dependencies from subfolders.
-
+The Dark Forest client deploy process is set up for players to easily fork the client and connect to
+the mainnet game, but that design decision currently makes it difficult to run a playable version
+of the game locally. This repository provides a setup for running a local game with just a few
+steps.
+
+## Builder's Guide
+
+For a full tutorial on using `darkforest-local` with multiple **quests**, check out the [Community Round Builder's Guide](builders_guide.md).
+
+## How darkforest-local Works
+
+This repo uses [submodules](.gitmodules) to pair the [Dark Forest Ethereum
+backend](https://github.com/darkforest-eth/eth) with the [Dark Forest TypeScript
+frontend](https://github.com/darkforest-eth/client) so you can launch a local game.
+
+It also uses Yarn, a package manager and that allows for multiple [workspaces](https://classic.yarnpkg.com/lang/en/docs/workspaces) to exist within a project.
+
+The workspaces allows each [submodule](.gitmodules) to have their own packages and configuration. 
+Yarn places all of the packages for each submodule in the top level `node_modules/` folder.
+
+## Requirements
+* Install [Yarn](https://classic.yarnpkg.com/en/docs/install)
+* Install `node >= 14` [nvm](https://github.com/nvm-sh/nvm)
+
+### Fastest Method for Running a Local Game
+1. Fork [darkforest-local](https://github.com/projectsophon/darkforest-local) to your Github repo.
+2. `git clone https://github.com/<your_name>/darkforest-local.git`
+3. `git submodule update --init --recursive --remote --merge`
+4. `yarn`
+5. `yarn start`
+
+### Better Method for Running a Local Game
```

Comment:
```suggestion
### If you plan to make changes to `darkforest-local`
```

Reply:
I link to Better Method for Running a Local Game in the builder's guide so will need to update this